ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-nightlies-postsubmits.yaml

```diff
@@ -696,8 +696,68 @@ postsubmits:
       timeout: 5h0m0s
     labels:
       ci.openshift.io/generator: prowgen
-    max_concurrency: 7
```

Comment:
I think it's ok that `max_concurrency` dropped from `7` to `1` for nightlies. Right?

Reply:
Yes.